### DISCUSSION TITLE
Importing data as CSV, JSON, etc

### DISCUSSION THREAD

**Discussion by pateljoel** [keystonejs/keystone#6138] [Questions]
I have an old Contentful CMS and would like to move to keystone, the issue is, I have lots of posts and content on my CMS that needs to be moved over.

Is it possible that keystonejs is able to import this data?

Thanks.

**Comment by gautamsi** [comment]
You have to write your own code to import data in onConnect event

**Comment by bladey** [comment]
@pateljoel Keystone doesn't have a data importer at the moment, but what @gautamsi has suggested would be the best way, on the `onConnect` event looping over your data and creating items in the DB.
